tests/osgi-tests/pom.xml

```diff
@@ -203,20 +202,51 @@
                             </execution>
                         </executions>
                     </plugin>
+
+                    <plugin>
+                        <artifactId>maven-resources-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <!--copying distribution/conf directory to the target folder-->
+                                <id>copy-resources</id>
+                                <phase>generate-resources</phase>
+                                <goals>
+                                    <goal>copy-resources</goal>
+                                </goals>
+                                <configuration>
+                                    <outputDirectory>${project.build.directory}/wso2carbon-kernel-${project.version}/conf/</outputDirectory>
+                                    <overwrite>true</overwrite>
+                                    <resources>
+                                        <resource>
+                                            <directory>src/test/resources/conf</directory>
```

Comment:
Entire conf folder is required here. So no use of coping each file individually. And there is underlying folder structure to be kept.